client/src/components/CustomCountBadge/index.tsx

```diff
@@ -0,0 +1 @@
+export { default as CustomCountBadge } from './CustomCountBadge';
```

Comment:
is this component used anywhere?

Reply:
Yes, it's used on Schedule page. I extracted it to separate component to reuse it on Mentor Dashboard page, but it was a bit early. I changed my mind and didn't add other components for now. However, I will keep it